src/main/resources/org/jenkinsci/plugins/pipeline/utility/steps/conf/ReadPropertiesStep/help.html

```diff
@@ -45,6 +45,11 @@
         An Optional Map containing default key/values.
         <em>These are added to the resulting map first.</em>
     </li>
+    <li>
+        <code>interpolate</code>:
+        Flag to indicate if the properties should be interpolated or not.
+        In case of error or cycling dependencies, the original properties will be returned.
```

Comment:
An example would be nice here. Not all users understands what _interpolated_ means :)

Reply:
Done! I've added an extra example with interpolation.